core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java

```diff
@@ -194,8 +192,12 @@ private ArtifactSignerProvider loadSigners(
               }
 
               slashingProtection.ifPresent(
-                  slashingProtection -> slashingProtection.registerValidators(validators));
-
+                  slashingProtection -> {
```

Comment:
raise a issue to check if we should prune JUST our data, or EVERYONE's data in the database?